.buildkite/pipeline.yml

```diff
@@ -94,6 +94,29 @@ steps:
                     - BUILDKITE_PULL_REQUEST=${BUILDKITE_PULL_REQUEST}
                     - GITHUB_PR_LABELS=${GITHUB_PR_LABELS}
 
+  - label: "Trigger Xpack/Winlogbeat"
+    plugins:
+      - monorepo-diff#v1.0.1:
+          diff: "git diff --name-only ${GITHUB_PR_TARGET_BRANCH}...HEAD"
+          watch:
+            - path:
+                - x-pack/winlogbeat/
+                - .buildkite/x-pack/pipeline.xpack.winlogbeat.yml
```

Comment:
I just thought that our tests can be affected by changes in hooks (like pre-command). So perhaps we should also add `.buildkite/hooks/` (we'll need to add it later for everything already migrated)

Reply:
Added it in the separate PR: https://github.com/elastic/beats/pull/38871